external-crates/move/crates/move-vm-runtime/src/unit_tests/invariant_violation_tests.rs

```diff
@@ -1,109 +0,0 @@
-// Copyright (c) The Move Contributors
```

Comment:
Was testing old check behavior that has been disabled for a couple years now, so this test was checking non-current-expected behavior and should have been removed long ago...